### PR TITLE
Add tips for generation with Int8 models

### PR DESCRIPTION
**Pull request by lewtun**
# What does this PR do?

Adds some tips to avoid gotchas with text generation and Int8 models

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

cc @gante @younesbelkada

### CODE REVIEW

File: docs/source/en/perf_infer_gpu_one.mdx [hunk 1]
```diff
@@ -36,20 +38,45 @@ Below are some notes to help you use this module, or follow the demos on [Google
 
 ### Requirements
 
-- Make sure you run that on NVIDIA GPUs that support 8-bit tensor cores (Turing, Ampere or newer architectures - e.g. T4, RTX20s RTX30s, A40-A100).
+- Make sure you run on NVIDIA GPUs that support 8-bit tensor cores (Turing, Ampere or newer architectures - e.g. T4, RTX20s RTX30s, A40-A100).
```
Comment (younesbelkada) [inline docs/source/en/perf_infer_gpu_one.mdx:41]: (unrelated to this PR but from the version `0.37.0` of `bitsandbytes` all GPU models should be supported, check: https://github.com/TimDettmers/bitsandbytes/releases/tag/0.37.0

**Review by younesbelkada — APPROVED** [review]
Thanks a lot for this! 💯

File: docs/source/en/perf_infer_gpu_one.mdx [hunk 2]
```diff
@@ -36,20 +38,45 @@ Below are some notes to help you use this module, or follow the demos on [Google
...
+```py
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+model_name = "bigscience/bloom-2b5"
+tokenizer = AutoTokenizer.from_pretrained(model_name)
+model_8bit = AutoModelForCausalLM.from_pretrained(model_name, device_map="auto", load_in_8bit=True)
+
+text = "Hello, my llama is cute"
+inputs = tokenizer(prompt, return_tensors="pt")
+inputs = {k:v.cuda() for k,v, in inputs.items()}
```
Comment (younesbelkada) [inline docs/source/en/perf_infer_gpu_one.mdx:74]: nit: you can also do:
```suggestion
inputs = tokenizer(prompt, return_tensors="pt").to("cuda")
```

Comment (lewtun) [reply docs/source/en/perf_infer_gpu_one.mdx:74]: oh much nicer!

**Comment by gante** [comment]
<3

File: docs/source/en/perf_infer_gpu_one.mdx [hunk 3]
```diff
@@ -19,10 +19,12 @@ We have recently integrated `BetterTransformer` for faster inference on GPU for
 
 ## `bitsandbytes` integration for Int8 mixed-precision matrix decomposition
 
-Note that this feature is also totally applicable in a multi GPU setup as well.
+<Tip>
+Note that this feature can also be used in a multi GPU setup.
+</Tip>
```
Comment (sgugger) [inline docs/source/en/perf_infer_gpu_one.mdx:24]:
```suggestion
<Tip>

Note that this feature can also be used in a multi GPU setup.

</Tip>
```

**Review by sgugger — APPROVED** [review]
Thanks for adding those!

**Pull request merged**